### ISSUE TITLE
Why the tab title show full url in browser ??

### ISSUE THREAD

**Issue by dc198689** [declandewet/vue-meta#103]
Hi, I am using your basic configuration in every component, but why does it show the full url in the tab title such as i refresh the page or route to another page. See the attached image below.

![path](https://user-images.githubusercontent.com/3169209/27281791-fa50e8a0-551f-11e7-95a9-97436d709f58.png)

In the component, i just these setting in export default

```
metaInfo () {
     return {
          title: this.title || 'Loading',
	  titleTemplate: '%s - VueJs'
     }
},
computed: {
     title () {
           return 'Login'
     }
}
```

How should i fixed it ? Thanks to help !

**Comment by mathieumagalhaes** [comment]
I have a similar situation, where I see no title (AKA full url) on first render (SSR), but when I change from page to page through the client vue-router it does render title and other meta tags.

My server entry includes:
`...
const { app, router } = createApp();
  const {
    title, htmlAttrs, bodyAttrs, link, style, script, noscript, meta,
  } = app.$meta().inject();
...`

TL;DR first render (ssr) does not render meta, client-router afterwards does.
